android/java/res/xml/brave_privacy_preferences.xml

```diff
@@ -132,5 +132,11 @@
             android:title="@string/send_p3a_analytics_title"
             android:summary="@string/send_p3a_analytics_summary"
             android:defaultValue="false" />
+        <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
+            android:key="brave_stats_usage_ping"
+            android:order="26"
```

Comment:
Everywhere above the order is 25, which is strange by itself, and here the order is 26 :) .
Order doesn't have  matter because the items are re-arranged in `BravePrivacySettings.NEW_PRIVACY_PREFERENCE_ORDER` , but it is fun, that it was changed here now

Reply:
Yeah, I didn't notice previous numbers, changed to 25 for consistency